composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.x-dev"
+            "dev-master": "3.x-dev"
         }
```

Comment:
Better to remove the branch alias config entirely from this branch. Be careful when you merge it up to master though (`--no-commit` and check changes)

Reply:
OK I'll remove it

So what should the branch alias be on the master branch?

Reply:
3 makes sense, but it doesn't need to be defined in this branch